content/sensu-go/6.2/observability-pipeline/observe-schedule/metrics.md

```diff
@@ -0,0 +1,365 @@
+
+| Prometheus         |      |
+---------------------|------
+output_metric_format | `prometheus_text`
+documentation        | [Prometheus Exposition Text][17]
+example              | {{< code plain >}}http_requests_total{method="post",code="200"} 1027 1395066363000{{< /code >}}
+
+## Enrich metrics with tags
+
+[Output metric tags][1] are custom tags you can apply to enrich the metric points produced by check output metric extraction.
+
+You can use [check token substitution][22] for the [value attribute][21] to include any event attribute in an output metric tag.
+For example, this tag will list the `event.time` attribute:
+
+{{< code shell >}}
+"output_metric_tags": [
+  {
+    "name": "instance",
+    "value": "{{ .time }}"
+  }
+]{{< /code >}}
+
+## Process extracted and tagged metrics
+
+Specify the event handlers you want to process your Sensu metrics in the check [`output_metric_handlers`][3] attribute.
+With these event handlers, you can route metrics to one or more databases for storing and visualizing metrics, like Elasticsearch, InfluxDB, Grafana, and Graphite.
+
+Many of our most popular metrics integrations for [time-series and long-term event storage][18] include curated, configurable quick-start templates to integrate Sensu with your existing workflows.
+You can also use [Bonsai][8], the Sensu asset hub, to discover, download, and share dynamic runtime assets for processing metrics.
+
+In check definitions, the `output_metrics_handlers` list for metrics event handlers is distinct and separate from the `handlers` list for status event handlers.
+Having separate handlers attributes allows you to use different workflows for metrics and status without applying conditional filter logic.
+The events reference includes an [example event with check and metric data][20].
+
+You do not need to add a mutator to your check definition to process metrics with an event handler.
+The [metrics attribute][5] format automatically reduces metrics data complexity so event handlers can process metrics effectively.
+
+## Example metric check
+
+This check definition collects metrics in Nagios Performance Data [format][9], enriches the metrics with [custom tags][1], and sends the collected and tagged metrics to the [Prometheus Pushgateway Handler][19] for long-term storage and visualization.
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+type: CheckConfig
+api_version: core/v2
+metadata:
+  annotations:
+  - slack-channel: '#monitoring'
+  labels:
+  - region: us-west-1
+  name: collect-metrics
+  namespace: default
+spec:
+  check_hooks: null
+  command: collect.sh
+  discard_output: true
+  env_vars: null
+  handlers: []
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  output_metric_format: nagios_perfdata
+  output_metric_handlers:
+  - prometheus_gateway
+  output_metric_tags:
+  - name: response_time_in_ms
+    value: '{{ .time }}'
```

Comment:
I'm not sure if this tag is a good example for Nagios format --> the Prometheus Pushgateway handler.

Also, I think I need to add another tag example that doesn't use token substitution.

Reply:
I agree that we should test this example to see if it works out the way we think. 👍